<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NidsForm</class>
 <widget class="QWidget" name="NidsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>预警类型 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxAlarm"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>协议类型 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProto"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableNids"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
